src/TwigComponent/src/ComponentAttributes.php

```diff
@@ -80,6 +80,16 @@ public function render(string $attribute): ?string
         return $value;
     }
 
+    public function prepend(string $attribute, string $value, string $separator = ' '): string
+    {
+        return trim(sprintf('%s%s%s', $value, $separator, $this->render($attribute)));
```

Comment:
Genuine question, are the "trim" necessary ?

Reply:
Nope not at all, I just dislike the attribute value being suffixed/prefixed by an empty space.

No problem for me to remove if desired.

Reply:
No it was really by curiosity